zyte_api/__main__.py

```diff
@@ -23,35 +23,52 @@ class DontRetryErrorsFactory(RetryFactory):
     retry_condition = retry_if_exception(_is_throttling_error)
 
 
-logger = logging.getLogger('zyte_api')
+logger = logging.getLogger("zyte_api")
 
 _UNSET = object()
 
 
-async def run(queries, out, *, n_conn, stop_on_errors, api_url,
-              api_key=None, retry_errors=True):
+def write_output(result, out, pbar):
+    json.dump(result, out, ensure_ascii=False)
+    out.write("\n")
+    out.flush()
+    pbar.update()
+
+
+async def run(
+    queries,
+    out,
+    *,
+    n_conn,
+    stop_on_errors,
+    api_url,
+    api_key=None,
+    retry_errors=True,
+    include_all_responses,
```

Comment:
Not giving it a default value would make the parameter mandatory, making the change backward-incompatible.

Also, maybe we could shorten it, e.g. `store_errors` / `--store-errors`.